packages/server/src/api/controllers/view/viewsV2.ts

```diff
@@ -31,10 +37,15 @@ export async function save(ctx: Ctx<ViewV2>) {
   }
 }
 
-export async function remove(ctx: Ctx) {
+export async function remove(ctx: Ctx<{}, {}>) {
```

Comment:
I think it could be better to use `Ctx<void, void>` when there is no request / response body, to be the `{}` indicates empty rather than none. It would also be ok just to leave them out and they'll fallback to `any`